README.md

```diff
@@ -2,7 +2,22 @@
 ## Overview
 This *Internet of Things Agent* is a bridge can be used to bridge between Telefonica's Thinking Things Closed protocol (TT from now on) and NGSI Context Brokers (like [Orion](https://github.com/telefonicaid/fiware-orion)). The Thinking Things protocol is a simplified protocol aimed to provide a simple platform to experiment with the Internet of Things.
 
-## Usage
+## Installation and usage
+### Using Docker
+If you are using Docker, you can download the latest Thinking Things module from Docker Hub, in order to try it. Do not use this installation mode for production purposes. 
+
+The Docker module has the prerequisite of having a Orion Context Broker that must be linked on start for the module to work. There is currently just one simple configuration offered for the IOTA, with in-memory transient storage (in the future, more configurations will be available).
+
+If there is a docker image running with the Context Broker and name `orion`, the following command will start a Thinking Things IoT Agent:
```

Comment:
This is a minor issue but maybe to avoid confusions, the text "If there is a docker image running with..." could be rephrased as "If there is a docker **container** running with..." Just to avoid the reader get confused about images and containers... :) But up to your discretion... ;)